core/trie/trie.go

```diff
@@ -3,13 +3,35 @@ package trie
 
 import (
 	"fmt"
+	"math/big"
 	"strings"
 
 	"github.com/NethermindEth/juno/core/crypto"
 	"github.com/NethermindEth/juno/core/felt"
 	"github.com/bits-and-blooms/bitset"
 )
 
+var (
+	maxHeight uint = 251
+	maxKey    *felt.Felt
+)
+
+type errMaxKey struct {
+	key *felt.Felt
+}
+
+func (e *errMaxKey) Error() string {
+	return fmt.Sprintf("key %s exceeds trie height %d", e.key.String(), maxHeight)
+}
+
+type errMaxHeight struct {
+	height uint
+}
+
+func (e *errMaxHeight) Error() string {
+	return fmt.Sprintf("max trie height is %d, got: %d", maxHeight, e.height)
+}
```

Comment:
We should follow [this](https://github.com/uber-go/guide/blob/master/style.md#error-types) convention for the errors here as well

Reply:
Yeah, we don't need these. We should use `fmt.Errorf`